MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.ComboBox.xaml

```diff
@@ -365,10 +365,10 @@
                                 <Condition Binding="{Binding IsMouseOver, RelativeSource={RelativeSource Self}}" Value="true" />
                                 <Condition Binding="{Binding IsEditable, RelativeSource={RelativeSource AncestorType={x:Type ComboBox}}}" Value="true" />
                             </MultiDataTrigger.Conditions>
-                            <Setter Property="BorderBrush" Value="{DynamicResource PrimaryHueMidBrush}" />
+                            <Setter Property="BorderBrush" Value="{Binding Path=(wpf:TextFieldAssist.HighlightedBrush), RelativeSource={RelativeSource AncestorType={x:Type ComboBox}}}" />
                         </MultiDataTrigger>
                         <Trigger Property="IsPressed" Value="true">
-                            <Setter TargetName="arrow" Property="Fill" Value="{DynamicResource PrimaryHueDarkBrush}" />
+                            <Setter TargetName="arrow" Property="Fill" Value="{Binding Path=(wpf:TextFieldAssist.HighlightedBrush), RelativeSource={RelativeSource AncestorType={x:Type ComboBox}}}" />
```

Comment:
This is not quite the same. The original code was using the `PrimaryHueDarkBrush` but switching it to this new brush would cause it to use the `PrimaryHueMidBrush` by default.

Reply:
My fault: I missed _dark_ in the name. I split the property into two and considered this special case with the dark color.